File: Assets/Scripts/MenuTextControl2.js
#pragma strict

function Start () {
    GetComponent.<Renderer>().material.color = Color.gray;
}

function OnMouseEnter () {
    GetComponent.<Renderer>().material.color = Color.blue;
}

function OnMouseExit () {
    GetComponent.<Renderer>().material.color = Color.gray;
}

function OnMouseUp () {
    // load the game
    Application.LoadLevel(2);
}

File: Assets/Scripts/MenuTextControl2.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.SceneManagement;

public class MenuTextControl2 : MonoBehaviour {
    void Start () {
        GetComponent<Renderer>().material.color = Color.gray;
    }

    void OnMouseEnter () {
        GetComponent<Renderer>().material.color = Color.blue;
    }

    void OnMouseExit () {
        GetComponent<Renderer>().material.color = Color.gray;
    }

    void OnMouseUp () {
        // load the game
        SceneManager.LoadScene("FutureCop");
    }
}
